tests/unit/mixins/authenticated-route-mixin-test.js

```diff
@@ -90,6 +91,14 @@ describe('AuthenticatedRouteMixin', () => {
         expect(route.transitionTo).to.have.been.calledWith(authenticationRoute);
       });
 
+      it('internally calls `triggerAuthentication` to accomplish the transition to the authentication route', function() {
+        let authenticationRoute = 'path/to/route';
+        route.set('authenticationRoute', authenticationRoute);
+
+        route.beforeModel(transition);
+        expect(route.triggerAuthentication).to.have.been.called;
+      });
```

Comment:
I don't think this needs to be tested. This is an internal implementation detail of the mixin that the test should not actually have to worry about.